bim2sim/sim_settings.py

```diff
@@ -605,6 +607,126 @@ def __init__(self):
                     "base for the selected construction set.",
         for_frontend=True,
     )
+    construction_class_walls = ChoiceSetting(
+        default='iwu_heavy',
+        choices={
+            'iwu_heavy': 'Wall structures according to iwu heavy standard',
```

Comment:
Please add a note which of these choices additionally require the year of construction for parametrization.

Reply:
fixed with afaf6f694fbadc8c52ca8cd5c5bf7dbc258ce124